test/mockFunctions.spec.js

```diff
@@ -14,8 +14,34 @@ O foco aqui é a utilização de mock functions.
 ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 */
 
+jest.mock('../src/mockFunctions');
+
 describe('verifica as funções e os mocks', () => {
   // Crie suas mock functions aqui
+  mockFunctions.add.mockImplementation((a, b) => a + b);
+  mockFunctions.subtract.mockImplementation((a, b) => a -b);
+  mockFunctions.multiply.mockImplementation((a, b) => a * b);
+  mockFunctions.divide.mockImplementation((a, b) => a / b);
+
+  mockFunctions.power.mockImplementation((a, b) => {
+    let pow = a;
+    if ( a === 0 && b === 0) {
+      pow = 1;
+    } else {
+      for (let i = 1; i < b; i += 1) {
+        pow = pow * a;
+      }
+    }
+      return pow;
+  });
```

Comment:
```suggestion
  mockFunctions.power.mockImplementation((a, b) => math.Pow(a, b))
```
existe uma função de math que faz todo o trabalho por vc simplifica muiito

Reply:
Verdade, esqueci dela. Grato pelo lembrete.